### PR TITLE
chore(skip-release): prepare 1.0.0

### PR DESCRIPTION
**Pull request by lstocchi**
Signed-off-by: Luca Stocchi <lstocchi@redhat.com>

### CODE REVIEW

**Review by jeffmaury — APPROVED** [review]
LGTM. Asking @mohitsuman to approve as well before we publish

File: README.md [hunk 1]
```diff
@@ -17,6 +17,36 @@ A JetBrains IntelliJ plugin for interacting with Tekton Pipelines. This plugin i
 
 This new release is based on v1beta1. Although it still supports v1alpha1 resources (resources, conditions), we do not support v1alpha1 version for resources that have v1beta1.
 
+### Debug task/taskrun
+
+A Task and/or a TaskRun can now be started in debug mode. The container where the taskrun will be executed
+will be kept running even when the run fails. A terminal connected to the container will be opened so you can 
+interact with it and identify any possible misbehavior/error.
+To use the debug feature you must have access to a kubernetes or openshift cluster with Tekton installed with a version
```
Comment (mohitsuman) [inline README.md:25]:
```suggestion
To use the debug feature, you must have access to a Kubernetes or OpenShift cluster with Tekton installed with a version 
```

**Review by mohitsuman — CHANGES_REQUESTED** [review]

**Review by mohitsuman — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -17,6 +17,36 @@ A JetBrains IntelliJ plugin for interacting with Tekton Pipelines. This plugin i
 
 This new release is based on v1beta1. Although it still supports v1alpha1 resources (resources, conditions), we do not support v1alpha1 version for resources that have v1beta1.
```
Comment (mohitsuman) [inline README.md:18]:
```suggestion
We support `v1beta1` API. Previous version `v1alpha1` may work, but we cannot guarantee that all features will work properly. If you have `v1alpha1` pipelines/tasks please use [migrating document](https://github.com/tektoncd/pipeline/blob/main/docs/migrating-v1alpha1-to-v1beta1.md) to migrate to `v1beta1`.
```

Comment (mohitsuman) [reply README.md:18]: @lstocchi This is to make it consistent with the information we have in VSCode tekton.

**Review by mohitsuman — CHANGES_REQUESTED** [review]

**Review by mohitsuman — APPROVED** [review]

**Pull request merged**